vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -211,10 +212,15 @@ bool TextViewManager::UpdateProperty(
       textBlock.ClearValue(xaml::Controls::TextBlock::LineStackingStrategyProperty());
     }
   } else if (propertyName == "selectable") {
-    if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Boolean)
-      textBlock.IsTextSelectionEnabled(propertyValue.AsBoolean());
-    else if (propertyValue.IsNull())
+    if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Boolean) {
+      const auto selectable = propertyValue.AsBoolean();
+      textBlock.IsTextSelectionEnabled(selectable);
+      if (selectable) {
+        EnsureUniqueTextFlyoutForXamlIsland(textBlock);
+      }
+    } else if (propertyValue.IsNull()) {
       textBlock.ClearValue(xaml::Controls::TextBlock::IsTextSelectionEnabledProperty());
```

Comment:
we should probably delete the custom type when clearing the value

Reply:
Sure, we can do that in this case.

Reply:
Done.

Reply:
Shouldn't we call `ClearUniqueTextFlyoutForXamlIsland` if selectable is set to false? Right now it's only called if it's set to null.